### DISCUSSION TITLE
这个绳网等级是什么？

### DISCUSSION THREAD

**Discussion by did1181** [share121/inter-knot#2598] [其他]
是GitHub等级吗
![18fb8aa5462309f7025dfc32340e0cf3d7cad617](https://github.com/user-attachments/assets/f986581d-ca3f-4c98-b62a-371b67405e6f)

**Comment by Qweulin** [comment]
是原神冒险等级<img src="https://img-static.mihoyo.com/communityweb/upload/fe198bb1523528c2b852d28626379a2a.png" alt="心情简单" width="75" >

**Comment by CreamAlphaWolfram** [comment]
可能指的是你在GitHub上的Repo个数吧？

**Comment by alanyagger** [comment]
是你在github上公开的reop数量

**Comment by ASTOU2311** [comment]
原来是这样的吗？
